test/services/GoodsItemCachingData.scala

```diff
@@ -31,7 +31,12 @@ trait GoodsItemCachingData {
   def decimalString(): String = Random.nextDouble().toString
 
   def createPackageInformation(): PackageInformation =
-    PackageInformation(Some(createRandomAlphanumericString(2)), Some(Random.nextInt(20) + 1), Some(createRandomAlphanumericString(150)))
+    PackageInformation(
+      createRandomAlphanumericString(5),
```

Comment:
Nit: This is also not adequate to the new implementation.

Reply:
In what way is it "inadequate"?    The id is a string.

Reply:
Sorry, by inadequate I mean that the id is created in a different way it is generated in the application code.